src/main/preload/mattermost.js

```diff
@@ -7,21 +7,18 @@
 /* eslint-disable no-magic-numbers */
 
 import {ipcRenderer, webFrame} from 'electron';
-import log from 'electron-log';
 
 import {NOTIFY_MENTION, IS_UNREAD, UNREAD_RESULT, SESSION_EXPIRED, SET_SERVER_NAME, REACT_APP_INITIALIZED, USER_ACTIVITY_UPDATE} from 'common/communication';
 
 const UNREAD_COUNT_INTERVAL = 1000;
 const CLEAR_CACHE_INTERVAL = 6 * 60 * 60 * 1000; // 6 hours
 
-Reflect.deleteProperty(global.Buffer); // http://electron.atom.io/docs/tutorial/security/#buffer-global
-
 let appVersion;
 let appName;
 let sessionExpired;
 let serverName;
 
-log.info('Initializing preload');
+console.log('Initializing preload');
```

Comment:
These changes are from a different PR, we should probably merge them separately.

Reply:
shouldn't that change have happened already in this branch? I'll remove it, but I wonder how it got there :/

Reply:
ok, so the actual commit doesn't have those changes, but something got wrong with the branch, I'll fix it and force push it.